src/extension.ts

```diff
@@ -447,6 +450,65 @@ class AmigaDebugExtension {
 		}
 	}
 
+	private async exe2adf(uri: vscode.Uri) {
```

Comment:
I think it would be better to just reuse 'shrinklerTerminal', rename it something generic like 'toolsTerminal' and not have 'exe2adf' be 99% the same as 'shrinkler'. could all be just 1 function.